onnx/defs/traditionalml/defs.cc

```diff
@@ -424,19 +430,20 @@ ONNX_ML_OPERATOR_SET_SCHEMA(
 
           // Load all values_* attributes.
           std::vector<std::string> values_strings;
-          bool values_strings_result = getRepeatedAttribute(
-              ctx, "values_strings", values_strings);
+          bool values_strings_result =
+              getRepeatedAttribute(ctx, "values_strings", values_strings);
           std::vector<int64_t> values_int64s;
-          bool values_int64s_result = getRepeatedAttribute(
-              ctx, "values_int64s", values_int64s);
+          bool values_int64s_result =
+              getRepeatedAttribute(ctx, "values_int64s", values_int64s);
           std::vector<float> values_floats;
-          bool values_floats_result = getRepeatedAttribute(
-              ctx, "values_floats", values_floats);
+          bool values_floats_result =
+              getRepeatedAttribute(ctx, "values_floats", values_floats);
 
           // Check if only one values_* attribute is set.
           if (static_cast<int>(values_strings_result) +
                   static_cast<int>(values_int64s_result) +
-                  static_cast<int>(values_floats_result) != 1)
+                  static_cast<int>(values_floats_result) !=
+              1)
```

Comment:
this is weird format. did you run "clang-format ..."

Reply:
Yes, I did. That's why we see a lot of formatting changes in this file.